x-pack/test/functional/apps/dashboard/reporting/README.md

```diff
@@ -0,0 +1,17 @@
+## The Reporting Tests
+
+### Reporting baseline snapshots
+
+The functional version of the reporting tests create a few pdf reports and do a snapshot comparison against a couple baselines.  The baseline images are stored in `./functional/reports/baseline`.
+
+#### Updating the baselines
+
+Every now and then visual changes will be made that will require the snapshots to be updated.  This is how you go about updating it.  I will discuss generating snapshots from chromium since that is the way of the future.
+
+1. Load the ES Archive containing the dashboard and data
+  ```
+  node scripts/es_archiver load reporting/ecommerce --config=x-pack/test/reporting/configs/chromium_functional.js
```

Comment:
Are these commands still up to date? Seems like the file got deleted

Reply:
Thanks for finding that. I should have updated the config path to the updated location. Done.